squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/html/scraper/HtmlScraper.java

```diff
@@ -47,7 +41,7 @@
     private Document doc;
     private Map<String,List<Triple>> staticMap = new HashMap<String,List<Triple>>();
     private Map<String,List<Triple>> selectedMap = new HashMap<String,List<Triple>>();
-
+    private Set<String> generatedUris = new HashSet<>();
```

Comment:
This is not working since the scraper has to be thread save. However, adding an attribute that is bound to a single call of  the `scrape(...)` method will break this!